example/lib/BUILD.bazel

```diff
@@ -1,10 +1,11 @@
-load("@aspect_bazel_lib//lib:copy_to_directory.bzl", "copy_to_directory")
+load("//js:js_package.bzl", "js_package")
```

Comment:
I think it ought to come from `defs.bzl` with `js_binary` and `js_test` WDYT? Is it likely to have extra `load` starlark deps that those rules don't?

Reply:
It depends on DeclarationInfo from rules_nodejs core